docs/tutorials/connecting-to-the-internet-via-wifi.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Setting up wifi on ev3
```

Comment:
Title should be `Setting up Wi-Fi Using Command Line Tools`.

Also, the file name needs to be changed to match the title (or something very close to it). `connecting-to-the-internet-via-wifi.md` will be reserved for a tutorial for setting up Wi-Fi on the EV3 itself (using brickman)